docs/DeveloperGuide.md

```diff
@@ -347,26 +347,40 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | user | add a new task|
+`* * *` | user | delete a task | remove tasks that I have completed.
+`* * *` | user | delete multiple task| remove multiple tasks that I have completed.
+`* * *` | user | edit a task | update relevant information about the task
+`* * *` | user | search for a task | see detailed information about the task.
+`* * *` | user | update the status of a task | view completed or pending tasks. 
+`* * *` | user | see upcoming tasks | see what needs to be done soon.
+`* *` | user | categorize a task | organize the tasks.
+`* *` | user | favourite tasks | keep track of important tasks.
+`* *` | user | see alerts on upcoming tasks | complete time sensitive tasks.
+`* *` | user | quickly add predefined tasks | to save time defining tasks
+`* *` | user | can undo last command | to fix mistakes.
+`* *` | user | be notified if two tasks have a time conflict | be aware of schedule conflicts
+`* *` | user | can use multiple devices | have more opportunity to check task manager
+`* *` | user | can undo last command | to fix mistakes.
+`*` | user | drag and drop events from UI | change event times easier
+`*` | user | copy and paste events from UI | add events quicker
+`*` | user | can send copy of task manager | share my schedule easily
+`*` | user | can have a shared group calendar | to coordinate with team members
 
 {More to be added}
 
 ## Appendix B : Use Cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Task Manager` and the **Actor** is the `user`, unless specified otherwise)
 
-#### Use case: Delete person
```

Comment:
Good to see you have use cases.
Remember you can follow the interaction in use cases to implement the feature (as actually this how use cases are supposed to work).